modules/nw-kuryr-cleanup.adoc

```diff
@@ -107,79 +98,95 @@ $ function REMFIN {
 The function takes two parameters: the first parameter is name of the resource, and the second parameter is the
 finalizer to remove. The named resource is removed from the cluster and its definition is replaced with copied data excluding the specified finalizer.
 
+. To remove Kuryr service `service-subnet-gateway-ip`, enter the following command:
++
+[source,terminal]
+----
+(venv) $ if $(oc get -n openshift-kuryr service service-subnet-gateway-ip 2>/dev/null >/dev/null); then
```

Comment:
How about using the &> redirection operator to redirect both stdout and stderr to /dev/null?
`oc get -n openshift-kuryr service service-subnet-gateway-ip &>/dev/null`

Reply:
good point. I'll change that.